GitHubCore.ps1

```diff
@@ -195,6 +195,7 @@ function Invoke-GHRestMethod
     }
 
     $NoStatus = Resolve-ParameterWithDefaultConfigurationValue -Name NoStatus -ConfigValueName DefaultNoStatus
+    $securityProtocol = [Net.ServicePointManager]::SecurityProtocol
```

Comment:
I should have mentioned this in a previous iteration.  When doing a backup/restore like this, the custom is to name it more clearly in order to help with self-documenting code.

```suggestion
    $originalSecurityProtocol = [Net.ServicePointManager]::SecurityProtocol
```

And then the similar name change in the `finally`

Reply:
Noted and updated